draft-ietf-jsonpath-base.md

```diff
@@ -1361,9 +1362,47 @@ Provisional registration? (standards tree only):
 
 # Security Considerations {#Security}
 
-This section gives security considerations, as required by {{RFC3552}}.
-
-
+Security considerations for JSONPath can stem from
+
+* attack vectors on JSONPath implementations, and
+* the way JSONPath is used in security-relevant mechanisms.
+
+## Attack vectors on JSONPath Implementations
+
+Historically, JSONPath has often been implemented by feeding parts of
+the query to an underlying programming language engine, e.g.,
+JavaScript.
+This approach is well known to lead to injection attacks and would
+require perfect input validation to prevent these attacks (see
+{{Section 12 of -json}} for similar considerations for JSON itself).
+Instead, JSONPath implementations need to implement the entire syntax
+of the query without relying on the parsers of programming language
+engines.
+
+Attacks on availability may attempt to trigger unusually expensive
+runtime performance exhibited by certain implementations in certain
+cases.
+(See {{Section 10 of -cbor}} for issues in hash-table implementations,
```

Comment:
I think it would be simpler to point to the security-considerations section or I-Regexp, which describes the use of toxic regexps to accomplish denial-of-service, and how the use of I-Regexp somewhat addresses that problem.